### PR TITLE
Corrects the attribution in the changelog for Ants to include Keelin.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Corrects the attribution in the changelog for Ants to include Keelin.

## Why It's Good For The Game

ants were designed by keelin over at goon, the goon port forgot to credit them for the design work, as such i have updated the changelog to properly credit them

### CODE REVIEW

File: html/changelogs/archive/2021-07.yml [hunk 1]
```diff
@@ -548,7 +548,7 @@
       or jump to a location and you wont jump back to the person you tracked
   Sheits:
   - bugfix: Flight potions now work on fly people, androids and skeletons
-  Wallem:
+  Keelin & Wallem:
   - bugfix: Lets people wash the ant debuff off of themselves, as intended.
   - rscadd: Due to a leak in Nanotrasen Brand Ant Farms during shipping, a large influx
       of space-ants have been inadvertently dumped onto every Nanotrasen station.
```
Comment (LemonInTheDark) [inline html/changelogs/archive/2021-07.yml:554]: The bugfix was technically just wallem, and the rscadd was the two of them. honestly not sure if it matters tho

Comment (Mothblocks) [reply html/changelogs/archive/2021-07.yml:554]: eh

**Review by Mothblocks — APPROVED** [review]

**Comment by optimumtact** [comment]
this doesn't remotely explain why this should be attributed to Keelin

**Comment by ExcessiveUseOfCobblestone** [comment]
Just remove this fricken reagent oh my god

**Comment by Iamgoofball** [comment]
@Wallemations can you explain i dont want to copypaste the conversation from the last 5 PRs and #coderbus and the coding general chat

**Comment by Wallemations** [comment]
> @Wallemations can you explain i dont want to copypaste the conversation from the last 5 PRs and #coderbus and the coding general chat

The ant PR _started off_ as a port of a goon chem, apparently there were some sort of rules against that which I wasn't entirely aware of, so a ton of people told me I needed to put credit to the guy who coded goon ants in the changelog.

I'm honestly just trying to not get over a month of my work removed bro I don't understand the semantics at _**all**_.

**Comment by LemonInTheDark** [comment]
> 
> 
> @Wallemations can you explain i dont want to copypaste the conversation from the last 5 PRs and #coderbus and the coding general chat

it should say something like, based off work done by or something

**Comment by Iamgoofball** [comment]
done

**Comment by optimumtact** [comment]
Please
for the LOVE OF GOD
FILL OUT THE PR TEMPLATE PROPERLY GOOF

YOU KNOW HOW THIS WORKS, JUST DO IT

**Comment by NanoCats** [comment]
> Just remove this fricken reagent oh my god

No, god save the ants, long may they reign.

**Pull request merged**